src/circuit/constants.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+const minute = 60 * 1000
+
+module.exports = {
```

Comment:
We should document these and clarify what each are for, some are clearer than others.

Reply:
I added comments and also updated the `CONFIGURATION.md` file.